Zend/zend.c

```diff
@@ -1175,20 +1175,24 @@ ZEND_COLD void zenderror(const char *error) /* {{{ */
 }
 /* }}} */
 
-ZEND_API ZEND_COLD ZEND_NORETURN void _zend_bailout(const char *filename, uint32_t lineno) /* {{{ */
+ZEND_API ZEND_COLD ZEND_NORETURN void _zend_bailout_without_gc_protect(const char *filename, uint32_t lineno)
```

Comment:
Maybe we could pick a more generic name, like `_zend_nonfatal_bailout` to make it more clear which is to be used.